### DISCUSSION TITLE
PHP Scripts to Accomodate

### DISCUSSION THREAD

**Discussion by MikeFairbrother** [ColorlibHQ/AdminLTE#3291] [General]
Hello

I am sorry if this has been asked before or I'm missing an obvious answer here. I am trying to do some work for a charity and my aim is to use AdminLTE for some backend database management, keeping track of cases they are dealing with etc. It's no major problem starting from scratch I just wondered if anyone knew of a PHP script to get me started such as login.php, register.php, users.php, change_password.php etc that is compatible with this amazing theme?

Thank you so much

**Comment by danny007in** [comment]
https://github.com/ColorlibHQ/AdminLTE/issues/2542 Check
